tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlDatabaseMonitoringTests.cs

```diff
@@ -0,0 +1,120 @@
+// <copyright file="MySqlDatabaseMonitoringTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    [Trait("RequiresDockerDependency", "true")]
+    public class MySqlDatabaseMonitoringTests : TracingIntegrationTest
+    {
+        public MySqlDatabaseMonitoringTests(ITestOutputHelper output)
+            : base("MySql", output)
+        {
+            SetServiceVersion("1.0.0");
+            SetEnvironmentVariable("DD_ENV", "testing");
+        }
+
+        public static IEnumerable<object[]> GetMySql8Data()
+        {
+            foreach (object[] item in PackageVersions.MySqlData)
+            {
+                if (!((string)item[0]).StartsWith("8") && !string.IsNullOrEmpty((string)item[0]))
+                {
+                    continue;
+                }
+
+                yield return item;
+            }
+        }
+
+        public override Result ValidateIntegrationSpan(MockSpan span) => span.IsMySql();
+
+        [SkippableTheory]
+        [MemberData(nameof(GetMySql8Data))]
+        [Trait("Category", "EndToEnd")]
+        public void SubmitDbmCommentedSpanspropagationDisabled(string packageVersion)
```

Comment:
It might also be good to test two other cases:
1. The default case, where environment variable `DD_DBM_PROPAGATION_MODE` is not set
2. Bad case, where environment variable `DD_DBM_PROPAGATION_MODE` is set to an unrecognized value